wordpress/plugins/region-report/region-report.js

```diff
@@ -609,10 +749,35 @@ class RegionReport {
     setupOverviewMap() {
         this.overviewMap = L.map(`region-report-overview-map-map-${this.postId}`, { maxZoom: 19, zoomControl: false });
         L.tileLayer("https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png").addTo(this.overviewMap);
+        this.overviewMap.createPane('main');
+        this.overviewMap.createPane('control');
+        this.overviewMap.getPane('main').style.zIndex = 800;
+        this.overviewMap.getPane('control').style.zIndex = 400;
         this.overviewMapHyperlink = document.getElementById(`region-report-overview-map-hyperlink-${this.postId}`);
     }
 
-    overviewMapAppState(layers, boundaryLayer, bounds, network, park) {
+    overviewMapAppState() {
+        let layers = [];
+
+        Object.values(this.minimapLayers).forEach(
+            layer => {
+                if (this.overviewMap.hasLayer(layer)) {
+                    layers.push(layer.metadata.layer);
+                }
+            }
+        );
+
+        layers.push(this.appBoundaryLayer);
+
+        switch (this.minimapState) {
+            case 'all':
+                layers = layers.concat(this.allLayers.metadata?.layers);
+                break;
+            case 'public':
+                layers = layers.concat(this.publicLayers.metadata?.layers);
+                break;
+        }
+
         return [
             "^ ",
             "~:autosave?",
```

Comment:
Jeebus, is this manually emitting transit-js?

Reply:
Umm, yes.
Sorry, it was the only solution I could come up with at the time, and I was clearly too lazy to make a utility of JSON to do the conversion
Now that I know what it is, I should use a package to do that.